### PR TITLE
CAF-2970: For Audit 3.1.0: Updated Features and Overview pages. Added…

### PR DESCRIPTION
**Pull request by CJComac**
… new blog post.

### CODE REVIEW

**Comment by glucy** [comment]
Dev Build: https://pages.github.hpe.com/sepg-docs-qa/CAFAudit_CI_audit-service_CAF-2970_Update_Overview_Features_and_Blog/

File: docs/pages/en-us/Features.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ User and system audit events are defined per application.
 The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis and across multiple tenants
 
 ## Auto-Generated Client Library
-The service includes an auto-generated client-side library for type safety.  This client library also makes it easier to send user and system audit events to Elasticsearch.
+The service includes an auto-generated client-side library for type safety.  This client library also makes it easier to send user and system audit events to the Audit Web Service or Elasticsearch.
```
Comment (mulhollandc) [inline docs/pages/en-us/Features.md]: I would suggest changing this to "This client library also makes it easier to send user and system audit events to Elasticsearch directly or via the Audit Web Service."

File: docs/pages/en-us/overview.md [hunk 2]
```diff
@@ -18,20 +18,30 @@ The Audit service provides applications with a reliable, high-performance soluti
...
 
 ## Introduction
-The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to Elasticsearch where they are indexed according to each tenant.
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to an endpoint such as the Audit Web Service API or directly to Elasticsearch where they are indexed according to each tenant.
```
Comment (mulhollandc) [inline docs/pages/en-us/overview.md]: In my opinion I think I'd drop the use of endpoint throughout. Applications send events either directly to Elasticsearch or via the Audit Web Service.

File: docs/pages/en-us/Features.md [hunk 3]
```diff
@@ -32,5 +32,8 @@ The service is designed for durability as Elasticsearch provides data redundancy
 
 ## High-Availability
 The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+
+## Audit Web Service API and Client
```
Comment (mulhollandc) [inline docs/pages/en-us/Features.md]: I could be over thinking here but should the REST API be considered a key feature? Maybe it is. If so I'd rename the heading here to something like REST API Interface or something like that to focus on the fact we have provided a REST API interface to send audit events through to Elasticsearch. The only reason I'm questioning this is that the REST API we have for the Job Service was never considered as a key feature so just raising this.

File: docs/pages/en-us/Features.md [hunk 4]
```diff
@@ -32,5 +32,8 @@ The service is designed for durability as Elasticsearch provides data redundancy
...
+Audit Web Service for logging application audit events through its RESTful interface which sends them through to Elasticsearch.
```
Comment (arogan92) [inline docs/pages/en-us/Features.md]: Maybe reword this to big it up as a feature?

"The Audit Web Service is effective for logging application events ... which forwards them through to Elasticsearch"

File: docs/pages/en-us/overview.md (hunk 2 above)
Comment (CJComac) [reply docs/pages/en-us/overview.md]: I have re-wrote this in the latest commit to read: "...The Audit service is multi-tenant aware.  Using the generated client-side auditing library, audited applications send audit event messages directly to Elasticsearch or the Audit Web Service API where they are indexed according to each tenant."

File: docs/pages/en-us/overview.md [hunk 5]
```diff
@@ -18,20 +18,30 @@ The Audit service provides applications with a reliable, high-performance soluti
...
 ## User and System Actions
+
 Applications define user and system actions in an audit event definition file.  The definition is utilized to generate a client-side auditing library which raises the defined audit events.
 
 For more details on the audit event definition file, see [Getting Started](Getting-Started).
 
 ## Audit Library
-The audit events definition file is used by a code generation plugin to auto-generate a client-side Java library, which provide type safety and sends audit event messages to Elasticsearch.
+
+The audit events definition file is used by a code generation plugin to auto-generate a client-side Java library, which provides type safety and sends audit event messages to an endpoint.
```
Comment (arogan92) [inline docs/pages/en-us/overview.md]: - I would put _utilized_ instead of _used_. 
- _enforces_ instead of _provides_.

File: docs/pages/en-us/Features.md (hunk 3 above)
Comment (CJComac) [reply docs/pages/en-us/Features.md]: In the latest commit I have changed the heading to "Audit Web Service" and removed the mention of RESTful API

File: docs/pages/en-us/Features.md (hunk 1 above)
Comment (CJComac) [reply docs/pages/en-us/Features.md]: Adjusted in the latest commit

**Review by arogan92 — CHANGES_REQUESTED** [review]

File: docs/pages/en-us/Features.md [hunk 6]
```diff
@@ -31,9 +31,9 @@ The service is extensively scalable.  Elasticsearch is designed with scalability
 The service is designed for durability as Elasticsearch provides data redundancy across it's highly scalable cluster.
 
 ## High-Availability
-The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.
 
-## Audit Web Service API and Client
-Audit Web Service for logging application audit events through its RESTful interface which sends them through to Elasticsearch.
+## Audit Web Service
```
Comment (mulhollandc) [inline docs/pages/en-us/Features.md]: Chris please run this by @glucy. I'm not sure Audit Web Service should the name of the key feature or whether it should be REST API Interface. The description should at least mention the use of a REST API interface to send audit events to Elasticsearch. Apart from this, I'm good with your changes.

**Review by mulhollandc — CHANGES_REQUESTED** [review]

File: docs/pages/en-us/Features.md (hunk 6 above)
Comment (CJComac) [reply docs/pages/en-us/Features.md]: Spoken with @glucy and we have agreed to rename the heading to REST API and to read as: RESTful API which allows audit event messages to be indexed within Elasticsearch.

Committed in the latest commit

File: docs/pages/en-us/overview.md (hunk 5 above)
Comment (CJComac) [reply docs/pages/en-us/overview.md]: Adjusted in the latest commit

**Review by arogan92 — APPROVED** [review]

**Review by mulhollandc — APPROVED** [review]
